meta-iotqa/conf/test/refkit-image-common-extensive.manifest

```diff
@@ -1 +1,3 @@
 # Extensive tests for common profile
+oeqa.runtime.bluetooth.comm_bt_command
```

Comment:
If you need `bluetoothctl` on some testing image, add `BAD_RECOMMENDATIONS_remove = "bluez5-client"` to your config.